<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QWidget" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>signup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 204, 255)</string>
  </property>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>400</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0.258, y1:0.136364, x2:1, y2:0.534, stop:0 rgba(170, 85, 255, 255), stop:0.577465 rgba(85, 0, 127, 255), stop:0.943662 rgba(255, 0, 255, 255))</string>
   </property>
   <property name="text">
    <string>Organisation</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>380</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0.258, y1:0.136364, x2:1, y2:0.534, stop:0 rgba(170, 85, 255, 255), stop:0.577465 rgba(85, 0, 127, 255), stop:0.943662 rgba(255, 0, 255, 255))</string>
   </property>
   <property name="text">
    <string>Anonymous</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : qlineargradient(spread:pad, x1:0.258, y1:0.136364, x2:1, y2:0.534, stop:0 rgba(170, 85, 255, 255), stop:0.577465 rgba(85, 0, 127, 255), stop:0.943662 rgba(255, 0, 255, 255))
</string>
   </property>
   <property name="text">
    <string>Personal</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>620</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:
qlineargradient(spread:pad, x1:0.258, y1:0.136364, x2:1, y2:0.534, stop:0 rgba(170, 85, 255, 255), stop:0.577465 rgba(85, 0, 127, 255), stop:0.943662 rgba(255, 0, 255, 255))</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/new/prefix1/C:/Users/Yasaman/Desktop/logo3.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
